.github/workflows/check-pr.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
       image: ubuntu:22.04
     strategy:
       matrix:
-        torch-version: ["2.5", "2.6"]
+        torch-version: ["2.5", "2.6", "2.7", "nightly"]
```

Comment:
NOTE: pytorch 2.7.0 release date is planed to 4/25, due to https://dev-discuss.pytorch.org/t/pytorch-release-2-7-0-call-for-features/2780.